sdk/core/azure-core/src/main/java/com/azure/android/core/util/Base64Util.java

```diff
@@ -14,14 +13,20 @@ private Base64Util() {
         // Empty constructor to prevent instantiation of this class.
     }
 
+    private static Base64Wrapper base64Wrapper = new Base64Android();
```

Comment:
This is for testing purposes only right? If not should this be atomic, or locked when modified, as this isn't completely thread-safe right now.

Reply:
It is only for testing purposes. I probably should add a clarifying comment though.

Reply:
Would it be better to move the interface and class below to their own files?

Reply:
Leaving them in this file is fine, I just didn't have a complete understanding of this when I wrote the comment.